extensions/crossbell/src/upload-file.tsx

```diff
@@ -0,0 +1,60 @@
+import { Clipboard, showToast, getSelectedFinderItems, showHUD, closeMainWindow } from "@raycast/api";
+import fs from "fs";
+import fsp from "fs/promises";
+import { ipfsUploadFile } from "crossbell/ipfs";
+import { fetch, FormData } from "node-fetch-native";
+import { showFailureToast } from "@raycast/utils";
+import { fileURLToPath } from "url";
+
+global.FormData = FormData;
+global.fetch = fetch;
+
+export default async function Command() {
+  try {
+    const selectedItems = await getSelectedFinderItems();
+    try {
+      closeMainWindow();
+      const filePaths = selectedItems.map((item) => item.path).filter((path) => fs.lstatSync(path).isFile());
+      if (filePaths.length > 0) {
+        const files = await Promise.all(
+          filePaths.map(async (path) => {
+            const data = await fsp.readFile(path);
+            return { name: path, data };
+          }),
+        );
+        console.log({ files });
+        const results = await Promise.all(
```

Comment:
1. I'm worried that there will be a huge load of files. Would a request queue be better? For example, use `p-limit`.
2. Should we use `Promise.allSettled` to handle failed requests?

Reply:
Oh yes, I will make a change here